### PR TITLE
Contact Validation Feature

### PR DESCRIPTION
**Pull request by HeDevedUp**
used JavaScript to write validation checks 
used HTML to write the markup 
used CSS to design the validation

### CODE REVIEW

File: js/contact-form.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (zieeco) [inline js/contact-form.js]: - [ ]  **[REQUIRED CHANGES]**

:point_right: **NOTE: Please, watch the video below to see your form in action**

- Please, kindly remove the unnecessary new lines added in your js file as it makes your codebase not clean. You should add a maximum of one new line per time between functions, and loops. event listeners. 
- According to the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md), Your form error message should show up near the submit button and not above the submit button. You can make it show just below the submit button.
Please, kindly fix this.
- A user cannot make a submission to your form. Please, kindly fix this also.
- Refer to the video below. :clinking_glasses: 

**YOUR FORM IN ACTION** :point_down: 

https://user-images.githubusercontent.com/59397159/168336872-aca1f7b9-fce7-4483-8a1b-afb5203ca88a.mp4

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @HeDevedUp , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Well done with the error message but please normally when it shows the error message, the form is not supposed to be sent. Please kindly fix it. prevent the form from being sent when the email contains upper case

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @HeDevedUp , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
linters passing :heavy_check_mark: 
Readme file descriptive :heavy_check_mark: 
Correct gitflow :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**